skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -43,13 +43,13 @@ fun compilerForTarget(os: OS, arch: Arch): String =
             Arch.Wasm -> "Unexpected combination: $os & $arch"
         }
         OS.Android -> "clang++"
-        OS.Windows -> "cl.exe"
+        OS.Windows -> "clang-cl.exe"
```

Comment:
The clang change in Skiko is better to extract to a separate PR, so we can easily bisect/revert in case of issues